builtin/aws/lambda/deployment.go

```diff
@@ -4,4 +4,8 @@ import (
 	"github.com/hashicorp/waypoint-plugin-sdk/component"
 )
 
+func (d *Deployment) URL() string {
+	return ""
```

Comment:
I think this should be optional so these no-ops don't have to be implemented (this is more of SDK feedback, but repeating it here) and also so we don't break any previous plugins.

Reply:
I'm not sure how we can make it optional to be honest, what would you suggest in this case?

Reply:
No problem! I think a separate interface would be the way to go (in the SDK):

```go
type DeploymentWithURL interface {
  Deployment

  URL() string
}
```